cmd/kube-proxy/app/server.go

```diff
@@ -58,6 +60,7 @@ func NewProxyServer() *ProxyServer {
 // AddFlags adds flags for a specific ProxyServer to the specified FlagSet
 func (s *ProxyServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(&s.BindAddress, "bind_address", "The IP address for the proxy server to serve on (set to 0.0.0.0 for all interfaces)")
+	fs.Var(&s.PublicAddress, "public_address", "The public IP address for exported services")
```

Comment:
I'll make it clearer.  Right now it is only used for externally load-balanced services that do packet-mangling (AWS & Rackspace I think).

But I could imagine that we will have a way to export a service without setting up a load balancer, in which case this would be the address used there also.

I tried covering both cases, but it ended up being unclear.  I will change to cover only today's use case, and we can tweak it later if the future use case becomes a real use case.